modules/client-handler/src/main/java/org/apache/ignite/client/handler/configuration/ClientHandlerConfigurationModule.java

```diff
@@ -38,4 +41,9 @@ public ConfigurationType type() {
     public Collection<RootKey<?, ?>> rootKeys() {
         return Collections.singleton(ClientConnectorConfiguration.KEY);
     }
+
+    @Override
+    public Set<Validator<?, ?>> validators() {
+        return Set.of(SslConfigurationValidatorImpl.INSTANCE);
```

Comment:
This validator is repeated in two modules, you shouldn't do that, please remove one of them